node-operators/block-producers/operational-tasks/.pages

```diff
@@ -2,4 +2,5 @@ title: Operational Tasks
 hide: false
 nav:
   - index.md
+  - upgrade-your-node.md
```

Comment:
Then here, you would do 'Upgrade Your Node': 'upgrade-your-node.md' to indicate that the title on the left nav should be 'Upgrade Your Node' which is enough since it's nested under the Block Producers section